Extension/src/LanguageServer/client.ts

```diff
@@ -2401,6 +2401,12 @@ export class DefaultClient implements Client {
             if (modifiedConfig.compileCommands) {
                 modifiedConfig.compileCommands = cppProperties.resolvePath(modifiedConfig.compileCommands, os.platform() === "win32");
             }
+
+            if (modifiedConfig.includePath) {
+                const resolvedIncludePath: string[] = [];
+                modifiedConfig.includePath.forEach((path: string) => {resolvedIncludePath.push(cppProperties.resolvePath(path, os.platform() === "win32")); });
```

Comment:
This could be simplified to 
`modifiedConfig.includePath = modifiedConfig.includePath?.map((path: string) => cppProperties.resolvePath(path, os.platform() === "win32"));`

However, I don't know if this would still apply after following Colen's recommended changes.